lib/net.cpp

```diff
@@ -15,6 +15,26 @@
 
 // https://wiki.openssl.org/index.php/SSL/TLS_Client
 
+static std::string socket_error(int err)
+{
+    static char msgbuf[256] = { '\0' };
+#ifdef _WIN32
+    FormatMessage(FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS,
```

Comment:
The return value of FormatMessage is not checked here, and it should be.

Reply:
Fixed.